src/Docker.DotNet/Models/ServiceFilter.cs

```diff
@@ -0,0 +1,62 @@
+using System;
+using System.Collections.Generic;
+
+namespace Docker.DotNet.Models
+{
+    public class ServicesListParameters
+    {
+        public ServiceFilter Filters { get; set; }
+    }
+
+    public class ServiceFilter : Dictionary<string, string>
+    {
+        public string Id
+        {
+            get => this["id"];
+            set
+            {
+                if (ContainsKey("id"))

```

Comment:
I'm surprised you have to make this check. Why cant set just be `this["id"] = value;`